src/App.tsx

```diff
@@ -1,62 +1,149 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
+import { Good } from './react-app-env';
 
 export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  { name: 'Dumplings', id: 1 },
+  { name: 'Carrot', id: 2 },
+  { name: 'Eggs', id: 3 },
+  { name: 'Ice cream', id: 4 },
+  { name: 'Apple', id: 5 },
+  { name: 'Bread', id: 6 },
+  { name: 'Fish', id: 7 },
+  { name: 'Honey', id: 8 },
+  { name: 'Jam', id: 9 },
+  { name: 'Garlic', id: 10 },
 ];
```

Comment:
This can be done using the following methods😅

`goodsFromServer.map((name, id) => ({ name, id }))`